apps/src/calc/calc.js

```diff
@@ -143,16 +154,9 @@ Calc.init = function(config) {
         config.forceInsertTopBlock);
     }
 
-    appState.targetExpressions = generateExpressionsFromBlockXml(solutionBlocks);
+    appState.targetSet = generateEquationSetFromBlockXml(solutionBlocks);
```

Comment:
My issue with change this name is that it's used a lot, so I'd prefer less verbosity. It seems like searching for targetSet/userSet should make it reasonably easy to figure out that this is an EquationSet.